src/views/login/loginView.js

```diff
@@ -0,0 +1,82 @@
+const LoginPage = function() {
+return `
+${LoginContainer()};
+${SubmitFeedbackSection()};
+${SubmitFeedbackResults()};
+${Footer()};
+`
+}
+
+const LoginContainer = function(){
+    return`<section class="feedback grid">
+    ${LoginHeader()}
+    ${LoginForm()}
+    <div class="clearfix"></div>
+    </sections>
+    `
+}
+
+const LoginForm = function(){
+    return `
+    <form class="login-form" method="POST" action="users.php">
+        <input type="text" name="userName" placeholder="User Name"> <br>
```

Comment:
Unacceptable! Using `<br>`s for styling after a full week of training? This. Shall. Not. Pass! :stuck_out_tongue: